docker/run.sh

```diff
@@ -1,3 +1,3 @@
 sudo docker run -d --name=EB-mysql --env=MYSQL_ROOT_PASSWORD=mypassword mysql:5.7
-echo -e "EB-mysql\nroot\nmypassword\nblackboard" > ../mysql_auth.txt
```

Comment:
I think a better solution to solve the `-e` option is to check current OS while running this script.
Ref: https://stackoverflow.com/questions/3466166/how-to-check-if-running-in-cygwin-mac-or-linux/18790824